modules/ROOT/pages/insight.adoc

```diff
@@ -25,45 +23,38 @@ Insight helps you answer questions about your integrated apps, such as:
 * What was the result of a transaction?
 * If something went wrong during processing, at what point did the failure occur?
 
-Using the Insight Dashboard, you can view events that occur within the flows that handle your business transactions. Further, you can drill down into event-related data to access information that can help you analyze the root cause of failures, isolate performance bottlenecks, and test for compliance with company procedures.
+Use the Insight Dashboard to view events that occur within the flows that handle your business transactions. You can drill down into event-related data to access information that can help you analyze the root cause of failures, isolate performance bottlenecks, and test for compliance with company procedures.
 
 [IMPORTANT]
 ====
-Enabling Insight may cause a performance impact when processing application data. We strongly discourage enabling Insight in production environments for long periods of time. To monitor a production environment, we recommend using the tools listed in the xref:monitoring.adoc[monitoring section].
+When Insight is enabled, there is a performance impact when application data is processed. Do not enable Insight for long periods of time for production environments. To monitor a production environment, use the tools listed in the xref:monitoring.adoc[monitoring section].
```

Comment:
When Insight is enabled, data processing performance is affected. (I hate "impact" and getting rid of passive voice).